File: Volume 10 (1000 - 1099)/1074/1074.py
for i in range(int(input())):
    n = int(input())
    if n > 0 and n % 2 == 0: print("EVEN POSITIVE")
    elif n > 0 and n % 2 != 0: print("ODD POSITIVE")
    elif n < 0 and n % 2 == 0: print("EVEN NEGATIVE")
    elif n < 0 and n % 2 != 0: print("ODD NEGATIVE")
    else: print("NULL")

File: Volume 10 (1000 - 1099)/1074/1074.c
#include <stdio.h>

int main() {
    long int n, x;
    scanf("%ld", &n);
    while (n--) {
        scanf("%ld", &x);
        if (x == 0) printf("NULL\n");
        else if (x % 2 == 0) {
            if (x > 0) printf("EVEN POSITIVE\n");
            else printf("EVEN NEGATIVE\n");
        }
        else {
            if (x > 0) printf("ODD POSITIVE\n");
            else printf("ODD NEGATIVE\n");
        }
    }
    return 0;
}